pkgs/by-name/ss/ssh3/package.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, libxcrypt
+}:
+
+buildGoModule rec {
+  pname = "ssh3";
+  version = "0.1.4";
+
+  src = fetchFromGitHub {
+    owner = "francoismichel";
+    repo = "ssh3";
+    rev = "v${version}";
+    hash = "sha256-0bd2hdvgapTGEGM7gdpVwxelN5BRbmdcgANbRHZ/nRw=";
+  };
+
+  buildInputs = [
+    libxcrypt
+  ];
+
+  vendorHash = "sha256-ZtKxAKNyMnZ8v96GUUm4EukdIJD+ITDW9kHOez7nYmg=";
+
+  ldflags = [ "-s" "-w" ];
+
+  meta = with lib; {
+    description = "Revisit of the SSH protocol using QUIC + TLS 1.3 and HTTP authorization";
```

Comment:
```suggestion
    description = "SSH3: faster and rich secure shell using HTTP/3";
```

Reply:
I don't recommend adopting upstream "faster and rich secure shell" representation. This is a preprint paper headline, not a faithful description of what it is right now.

Reply:
Fair enough!